<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDesigWin</class>
 <widget class="QMainWindow" name="ReportDesigWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionVista_Previa"/>
    <addaction name="separator"/>
    <addaction name="actionConfigurar_pagina"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTexto"/>
   <addaction name="actionParametro"/>
   <addaction name="actionCampo"/>
   <addaction name="actionCampo_Relacional"/>
   <addaction name="actionCodeBars"/>
   <addaction name="separator"/>
   <addaction name="actionRectangulo_redondeado"/>
   <addaction name="actionLinea"/>
   <addaction name="actionImagen"/>
   <addaction name="separator"/>
   <addaction name="actionBorrar"/>
   <addaction name="separator"/>
   <addaction name="actionAbrir"/>
   <addaction name="actionGuardar"/>
   <addaction name="separator"/>
   <addaction name="actionAbrir_desde_BD"/>
   <addaction name="actionGuardar_en_BD"/>
   <addaction name="separator"/>
   <addaction name="actionVista_Previa"/>
  </widget>
  <action name="actionRectangulo_redondeado">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/rounded-rectangle.png</normaloff>:/icons/rounded-rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangulo redondeado</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar Archivo</string>
   </property>
  </action>
  <action name="actionTexto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/text-shadow.png</normaloff>:/icons/text-shadow.png</iconset>
   </property>
   <property name="text">
    <string>Texto</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Abrir Archivo</string>
   </property>
  </action>
  <action name="actionLinea">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/line.png</normaloff>:/icons/line.png</iconset>
   </property>
   <property name="text">
    <string>Linea</string>
   </property>
  </action>
  <action name="actionCodeBars">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/barcode.png</normaloff>:/icons/barcode.png</iconset>
   </property>
   <property name="text">
    <string>Codigo de Barras</string>
   </property>
  </action>
  <action name="actionImagen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/picture.png</normaloff>:/icons/picture.png</iconset>
   </property>
   <property name="text">
    <string>Imagen</string>
   </property>
  </action>
  <action name="actionBorrar">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Delete.png</normaloff>:/icons/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </action>
  <action name="actionConfigurar_pagina">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/config.png</normaloff>:/icons/config.png</iconset>
   </property>
   <property name="text">
    <string>Configurar pagina</string>
   </property>
  </action>
  <action name="actionCampo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/field.png</normaloff>:/icons/field.png</iconset>
   </property>
   <property name="text">
    <string>Campo</string>
   </property>
  </action>
  <action name="actionCampo_Relacional">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/field_rel.png</normaloff>:/icons/field_rel.png</iconset>
   </property>
   <property name="text">
    <string>Campo Relacional</string>
   </property>
  </action>
  <action name="actionVista_Previa">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/print-preview.png</normaloff>:/icons/print-preview.png</iconset>
   </property>
   <property name="text">
    <string>Vista Previa</string>
   </property>
  </action>
  <action name="actionAbrir_desde_BD">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/upload_db.png</normaloff>:/icons/upload_db.png</iconset>
   </property>
   <property name="text">
    <string>Abrir desde BD</string>
   </property>
  </action>
  <action name="actionGuardar_en_BD">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/save_db.png</normaloff>:/icons/save_db.png</iconset>
   </property>
   <property name="text">
    <string>Guardar en BD</string>
   </property>
  </action>
  <action name="actionParametro">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/text-shadow.png</normaloff>:/icons/text-shadow.png</iconset>
   </property>
   <property name="text">
    <string>Parametro</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
